src/middlewares/stats.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable require-jsdoc */
+
+import jwt from 'jsonwebtoken';
+
+import StatsService from '../services/db.service';
+import { notFound, error } from '../helpers/responses';
+
+const { getStat } = StatsService;
+
+class statsWare {
+  static async checkStats(req, res, next) {
```

Comment:
can you please use auth middleware

Reply:
For the same reason above, the auth middleware as is will interrupt requests and return auth responses. All we need here is a transparent check